src/lib.rs

```diff
@@ -94,15 +94,15 @@ fn AppRoutes() -> impl IntoView {
           <ProtectedRoute
             path=path!("login")
             view=LoginPage
-            redirect_path=move || ""
-            condition=user_is_logged_in
+            redirect_path=move || "/"
```

Comment:
Turns out if you just use a space for this, it will only redirect to the root path relative to the current path, meaning `localhost:1237/login` will keep redirecting back to `localhost:1237/login`. Using the slash guarantees it will redirect back to `localhost:!237` as intended.